### ISSUE TITLE
[ARPA P&E Reporting] Audit Report 

### ISSUE THREAD

**Issue by usdr-grants** [usdigitalresponse/usdr-gost#1349]
**Submitted Description:**
Include all expenditure data reported by project ID, and by subaward #, and by quarter - to assist State's in finding calculation issues as reports are loaded into the Reporter (instead of waiting on Treasury validations) and to assist in reconciling projects to our accounting system. This is Ohio's #1 priority!

**(Optional) Uploaded file - click to download:**

**Comment by as1729** [comment]
Closing in favor of #1303
